.github/workflows/ci.yml

```
@@ -5,6 +5,9 @@ on:
       - master
     tags: '*'
   pull_request:
+  schedule:
+    # Run CI against `master` every Sunday
+    - cron: '* * * * 0'
```

This is every minute on Sunday! I think it should be something like:
```suggestion
    - cron: '0 0 * * 0'
```